src/main/kotlin/com/github/insanusmokrassar/TelegramBotAPI/types/passport/EncryptedPassportElement.kt

```diff
@@ -0,0 +1,29 @@
+package com.github.insanusmokrassar.TelegramBotAPI.types.passport
+
+import com.github.insanusmokrassar.TelegramBotAPI.types.*
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class EncryptedPassportElement (
+    @SerialName(typeField)
+    val type: String,
+    @SerialName(dataField)
+    val data: Base64?,
```

Comment:
All fields here must be private. For each type of element must be created separated type and this type must be resolved according to the type field. You can look at the `RawMessage` to get an example